code/game/objects/items/spear.dm

```diff
@@ -181,8 +181,25 @@
 	desc = "A haphazardly-constructed yet still deadly weapon. The pinnacle of modern technology."
 	force = 12
 	throwforce = 22
-	armour_penetration = 15 //Enhanced armor piercing
+	reach = 2 //SKYRAT EDIT
+	armour_penetration = 15
+
+//SKYRAT EDIT ADDITION BEGIN - Slight antimob spear buff
+	var/faction_bonus_force = 8 //Bonus force dealt against certain factions. Blatantly stolen from the arrow code, because shitcode
```

Comment:
I don't think that's necessarily a great change here, overall it just makes the spear *even* stronger for the purpose of defeating bosses and mobs (20 damage unwielded, compared to 20 damage wielded before).

Reply:
Changed that, nerfing the unwielded damage slightly. It's now 16 unwielded and 24 wielded, which... might be okay, as other things have either better damage per hit (buffed bone axe) or DPS (tweaked bone sword) at the cost of risking getting hit.